jupyter_server/gateway/managers.py

```diff
@@ -290,6 +294,7 @@ async def list_kernel_specs(self):
         response = await gateway_request(kernel_spec_url, method="GET")
         kernel_specs = json_decode(response.body)
         kernel_specs = self._replace_path_kernelspec_resources(kernel_specs)
+        self.log.debug(f"Retrieved list of kernel specs for the uri: {kernel_spec_url}")
```

Comment:
This seems redundant since we're logging the same information prior to the gateway request.  Given this occurs at a (lab-induced) polling interval of every 10 seconds and an exception will be raised to indicate a failure, I'd prefer we remove this statement and go with a "no news is good news" strategy here and eliminate this extra noise.  Does that sound okay?
```suggestion
```

Reply:
:+1: Removed.